exe/qless-config

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env ruby
+
+require 'json'
+require 'qless'
+require 'thor'
+
+class Configurator < Thor
+
+  class_option :redis, :default => 'redis://localhost:6379/0'
```

Comment:
Looks like you could benefit from a run through rubocop.  Usually this is written as `default: 'redis://localhost:6379/0'`.